src/celldega/pre/__init__.py

```diff
@@ -952,4 +953,4 @@ def _check_required_files(technology, data_dir):
         )
 
 
-__all__ = ["landscape", "trx_tile", "boundary_tile"]
\ No newline at end of file
+__all__ = ["landscape", "trx_tile", "boundary_tile", "distance"]
```

Comment:
we do not need to expose "distance" in the pre module as far as I know.

Reply:
fixed.